robottelo/performance/perf_thread.py

```diff
@@ -0,0 +1,92 @@
+"""Test utilities for multi-threading programming"""
+from robottelo.performance.perf_candlepin import CandlepinPerfTest
+
+import logging
+import threading
+import time
+
+
+class PerfThread(threading.Thread):
+    def __init__(self, threadID, threadName, time_result_dict):
+        threading.Thread.__init__(self)
+        self.threadID = threadID
+        self.threadName = threadName
+        self.time_result_dict = time_result_dict
+        self.logger = logging.getLogger('robottelo')
+
+
+class DeletePerfThread(PerfThread):
+    def __init__(self, threadID, threadName, sublist, time_result_dict):
+        super(DeletePerfThread, self).__init__(
+            threadID, threadName, time_result_dict)
+        self.sublist = sublist
+
+    def run(self):
+        time.sleep(5)
+        self.logger.info('---------------------------------')
+        self.logger.debug('Start timing in thread {}'.format(self.threadID))
+        self.logger.info('---------------------------------')
```

Comment:
`info`, `debug`, `info`? If I have my debugger level set to `info`, won't I see a whole lot of this in my log file?

    ---------------------------------
    ---------------------------------
    ---------------------------------
    ---------------------------------
    ---------------------------------
    ---------------------------------
    ---------------------------------
    ---------------------------------